fundamentals/script.js

```diff
@@ -0,0 +1,274 @@
+const colorInput = document.querySelector('.color-selector');
+const widthInput = document.querySelector('.width-slider');
+const heightInput = document.querySelector('.height-slider');
+const colorBlock = document.querySelector('.color-block');
+const blockControlsWrapper = document.querySelector('.block-controls-wrapper');
+
+const TEST_DATA = {
+  ARRAY_OF_INTS: [1, 5, 6, 7]
+}
+
+var student = {
+  NAME : "David Rayy",
+  SCLAS : "VI",
+  ROLLNO : 12
+};
+
+/**
+ * TASK 1
+ *
+ */
+function getBiggerNumber(int1, int2) {
+  if ((int1 >= 40 && int1 <= 60) && (int2 >= 40 && int2 <= 60)) {
+    return int1 > int2 ? int1 : int2;
+  }
+
+  return 0;
+}
+
+/**
+ * TASK 2
+ *
+ */
+function getArraySum(intArr) {
+  return intArr.reduce(function (accumulator, currentValue) {
+    return accumulator += currentValue;
+  }, 0);
+}
+
+/**
+ * TASK 3
+ *
+ */
+function colorMixer() {
+  myColor = ["Red", "Green", "White", "Black"];
+  let output = myColor.join(',');
+}
+
+/**
+ * TASK 4
+ *
+ */
+function listProperties() {
+  for (let key in student) {
+    if (student.hasOwnProperty(key)) {
+      returnString += key + ',';
+    }
+  }
+}
+
+var stringToFormat = ' asd asd jjj bbb jjj '; // Global on purpous
+/**
+ * TASK 5
+ *
+ */
+function stripLeadTrail() {
+  stringToFormat.trim();
+}
+
+/**
+ * TASK 6
+ *
+ */
+function removeFirst() {
+  stringToFormat.replace('jjj','');
+}
+
+/**
+ * TASK 7
+ *
+ */
+function removeFirst() {
+  stringToFormat.search('jjj');
+}
+
+/**
+ * TASK 8
+ * '<p><strong><em>PHP Exercises</em></strong></p>' -- test string
+ */
+function strip_html_tags(string) {
+
+  let regExHTML = /<[^>]*>/g;
+  return string.replace(regExHTML, '');
+}
+
+/**
+ * TASK 9
+ * 'Hello world, welcome to the universe.' -- test string
+ */
+function removeFirst(inputString, end) {
+  inputString.endsWith(end);
+}
+
+/**
+ * TASK 10
+ *
+ */
+function removeFirst(inputObject) {
+  Object.keys(inputObject).length;
+}
+
+/**
+ * TASK 11
+ *
+ */
+function first(array, n) {
+  if (n <= 0) {
+    return array[0];
+  }
+  array.splice(n, array.length-1);
+  return array;
+}
+
+/**
+ * TASK 12
+ *
+ */
+function removeFirst(inputObject) {
+  var arr1=[3, 'a', 'a', 'a', 2, 3, 'a', 3, 'a', 2, 4, 9, 3];
+
+
+
+  arr1.forEach(element => {
+    if (itterObject.hasOwnProperty(element)) {
+      itterObject[element] += 1;
+    } else {
+      itterObject[element] = 1;
+    }
+  });
+
+  Object.keys(itterObject).forEach(key => {
+    let currentValue = itterObject[key];
+
+    if (currentValue > maxCount) {
+      maxCount = currentValue;
+      result = key + ': ' + currentValue;
+    }
+  });;
+}
+
+/**
+ * TASK 13
+ *
+ */
+function fizzBuzz() {
+  for (let i = 1; i <= 100; i++) {
+
```

Comment:
An unnecessary empty line. Please format this whole function. Its readability is really poor. You could also use `else-if` statements to ensure that you don't always check every scenario when it can only enter in one.

Reply:
This hasn't been resolved